engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version.html

```diff
@@ -67,6 +67,12 @@ <h3 translate>
         </a>
       </li>
 
+      <li ng-class="{active: isState('content-views.details.version.docker')}">
+        <a ui-sref="content-views.details.version.docker({versionId: version.id})">
+          <span translate>Docker Content</span>
```

Comment:
Similarly - this will show me docker repositories right? not the docker content itself?